### DISCUSSION TITLE
Activity Feed Module

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#1738] [Ideas]
We have a client with a feature requirement for an activity feed.  It's an inventory app, and they would like to see who/when/what changes for the inventory items.  We based the inventory app on Oqtane and think that an activity feed module would be interesting for any context.  Once we have something working in for our project, I'll start an OSS version as an Oqtane Module.

**Comment by lanthonyneville** [comment]
So in your case, the activity is I guess inventory transactions - pluses and minuses of stock, something like that? But I guess you are saying the module has been made generic to show any kind of events happening over time? Am I on the right track? 
Sounds interesting!

**Comment by hishamco** [comment]
We have a client with a feature requirement for an activity feed

> You can build it as a custom module, make is OSS will be a BIG plus
